oh_staff_ui/management/commands/import_projectitems.py

```diff
@@ -82,12 +84,12 @@ def add_projectitem(self, pi_dict: dict, has_parent=True) -> list:
         p.save()
 
     def format_date(self, date: str) -> str:
-        return datetime.datetime.strptime(date, "%d-%b-%y").strftime("%Y-%m-%d")
+        return datetime.datetime.strptime(date, "%Y-%m-%d %H:%M:%S").strftime("%Y-%m-%d")
```

Comment:
The additional time specification is unnecessary, as the model defines `ProjectItem.create_date` and `ProjectItem.last_modified_date` as `models.DateField` - any time detail is lost:
```
>>> ProjectItem.objects.get(pk=10259).create_date
datetime.date(2022, 9, 6)
```
If this is a concern, we should revise the model to use `DateTimeField` instead.

Reply:
Updated DB field to `DateTimeField`. Form display remains the same for these fields, this level of precision on the display side is not required.